apps/api/src/app/inbox/usecases/get-notifications/get-notifications.command.ts

```diff
@@ -30,4 +30,8 @@ export class GetNotificationsCommand extends EnvironmentWithSubscriber implement
   @IsOptional()
   @IsBoolean()
   readonly archived?: boolean;
+
+  @IsOptional()
+  @IsBoolean()
+  readonly snoozed?: boolean;
```

Comment:
this one should be already defined because we set it by the class transformer at the DTO level.
```suggestion
  @IsDefined()
  @IsBoolean()
  readonly snoozed: boolean;
```

Reply:
On the DTO level it will be transformed to boolean only if its defined, if you don't send the parameter at all, it will be undefined in DTO and here as well. It also makes sense because `snooze: false` !== `snooze: undefined`, in one case you want explicitely notification that are not snoozed in the other case you will get both snoozed and unsnoozed